openff/toolkit/_tests/test_examples.py

```diff
@@ -38,55 +36,70 @@ def run_script_str(script_str):
 
     """
     with tempfile.TemporaryDirectory() as tmp_dir:
-        temp_file_path = os.path.join(tmp_dir, "temp.py")
+        temp_file_path = pathlib.Path(tmp_dir, "temp.py").as_posix()
+
         # Create temporary python script.
         with open(temp_file_path, "w") as f:
             f.write(script_str)
         # Run the Python script.
         try:
             run_script_file(temp_file_path)
-        except:  # noqa
+        except Exception as error:
             script_str = textwrap.indent(script_str, "    ")
-            raise Exception(f"The following script failed:\n{script_str}")
+            raise Exception(f"The following script failed:\n{script_str}") from error
 
 
-def find_example_scripts():
+def find_example_scripts() -> list[str]:
     """Find all Python scripts, excluding Jupyter notebooks, in the examples folder.
 
     Returns
     -------
     example_file_paths : list[str]
         List of full paths to python scripts to execute.
     """
-    examples_dir_path = ROOT_DIR_PATH.joinpath("examples")
+    # Count on the examples/ path being equivalently accessible as the README file
+    readme_file_path = _get_readme_path()
+
+    if readme_file_path is None:
+        return list()
+
+    else:
+        examples_dir_path = pathlib.Path(_get_readme_path().parent, "examples")
 
-    # Examples that require RDKit
-    rdkit_examples = {
-        examples_dir_path.joinpath("conformer_energies/conformer_energies.py"),
-    }
+        # Examples that require RDKit
+        rdkit_examples = {
+            examples_dir_path / "conformer_energies/conformer_energies.py"
```

Comment:
Wow, trippy. I didn't realize that Path objects overrode the `/` operator. Made my day.